docs/chapters/04-1.md

```diff
@@ -0,0 +1,218 @@
+$$
+\begin{bmatrix}
+a_1^{(1)} & a_2^{(1)}  & a_3^{(1)}  & 0 & 0 & 0 & 0&\cdots  &0\\
+0 & a_1^{(1)}  & a_2^{(1)} & a_3^{(1)}  & 0&0&0&\cdots &0\\
+0 & 0 & a_1^{(1)} & a_2^{(1)}  & a_3^{(1)}  & 0&0&\cdots &0\\
+0 & 0 & 0& a_1^{(1)}  & a_2^{(1)}  &a_3^{(1)} &0&\cdots &0\\
+0 & 0 & 0& 0 & a_1^{(1)}  &a_2^{(1)} &a_3^{(1)} &\cdots &0\\
+\vdots&&\vdots&&\vdots&&\vdots&&\vdots
+\end{bmatrix}
+\begin{bmatrix}
+x_1\\
+x_2\\
+x_3\\
+x_4\\
+\vdots\\
+x_k\\
+\vdots\\
+x_n
+\end{bmatrix}
+$$
+Both the upper right part and lower left part of the matrix are filled with 0 due to the property of locality. Here, putting zeros around a matrix is called sparsity. And the replication of the certain kernel again and again is called weight sharing.
+
+### Multile Layers of Toeplitz Matrix
+After these changes, the number of parameters we have is 3 (i.e. $a_1,a_2,a_3$). Compared with the previous weight matrix, whose number of parameter is 12 (i.e. $w_{11},w_{12},\cdots,w_{43}$), the current parameters are too restrictive. So more parameters are needed.
+
+We can regard the previous matrix as one layer (i.e. a convolutional layer) with the kernel $\mathbf{a}^{(1)}$ and construct multiple layers with different kernels $\mathbf{a}^{(2)}$, $\mathbf{a}^{(3)}$, etc. Thus, the number of parameters increases. 
+
+Each layer has a matrix containing just one kernel that replicates multiple times. This kind of matrix is called toeplitz matrix. In every toeplitz matrix, each descending diagonal from left to right is constant. As for the toeplitz matrices that we use here, they are also sparse matrices.
+
+Given the first kernel $\mathbf{a}^{(1)}$ and the input vector $\mathbf{x}$, the first entry in the output given by this layer is $a_1^{(1)} x_1 + a_2^{(1)} x_2 + a_3^{(1)}x_3$. Therefore, the whole output vector is like
+$$
+\begin{bmatrix}
+\mathbf{a}^{(1)}x[1:3]\\
+\mathbf{a}^{(1)}x[2:4]\\
+\mathbf{a}^{(1)}x[3:5]\\
+\vdots
+\end{bmatrix}
+$$
+For other kernels (e.g. $\mathbf{a}^{(2)}$ and $\mathbf{a}^{(3)}$) in following convolutional layers, we can apply the same matrix multiplication method and get the similar results.
+
+
+## [Listening to Convolutions - Jupyter Notebook](0:33:01-0:59:59)
+
+The Jupyter Notebook can be found [here](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/07-listening_to_kernels.ipynb). Note the names of github repository and conda environment have changed.
+
+In this notebook, we are going to explore Convolution as 'running scalar product'.
+
+Importing library librosa and we can load data $\mathbf{x}$ and sampling rate. In this case, there are 70641 samples, learning rate is 22.05kHz and total time is 3.2s. The imported audio signal is wavy (refer to Fig1) and we can guess what it sounds like from the amplitude of y axis. Listening to x(t), the audio signal is actually the sound played when turning off the Windows system (refer to Fig2). 
+![Fig1 Audio signal](https://i.imgur.com/LLdvx8d.png)
```

Comment:
Images have to be submitted too!
Also, use `<center><img src="image_name.jpg" width="350px" /><br><b>Fig</b> Blah, blah</center>` format.